FCFS+ (#5)
thomas-fossati/draft-cf-reg-update: draft-fossati-core-cf-reg-update.md

# IANA Considerations {#iana}

The CoAP Content-Formats registration procedures defined in {{Section 12.3 of -coap}} are updated as follows:

| Range | Registration Procedures |
|--------|--------|
| 0-255 | Expert Review (Full) |
| 256-9999 | IETF Review or IESG Approval |
| 10000-64999 | Expert Review (Expert Check: FCFS+) |
| 65000-65535 | Experimental use (no operational use) |
{: #tbl-new-cf-proc title="Updated CoAP Content-Formats Registration Procedures"}

> Where the "FCFS+" expert check comprises the following steps:

Author: ```suggestion
The DE checks consist of the following steps:
```
